runtimevar/blobvar/example_test.go

```diff
@@ -63,3 +63,19 @@ func Example() {
 	// Output:
 	// foo.com running on port 80
 }
+
+func Example_openVariable() {
+	// OpenVariable creates a *runtimevar.Variable from a URL.
+	// This example watches a variable based on a file-based blob.Bucket with JSON.
+	ctx := context.Background()
+	v, err := runtimevar.OpenVariable(ctx, "blob://myvar.json?bucket=file:///mypath&decoder=json")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	snapshot, err := v.Watch(ctx)
+	if err != nil {
+		log.Fatal(err)
+	}
+	_, _ = snapshot, err
```

Comment:
Don't need the `, err` part since you've already handled it.

Reply:
Removed the handling, it's not important to the example.